src/Cli/func/Common/TemplatesManager.cs

```diff
@@ -69,7 +69,10 @@ private static async Task<IEnumerable<Template>> GetTemplates()
             var extensionBundleManager = ExtensionBundleHelper.GetExtensionBundleManager();
             string templatesJson;
 
-            if (extensionBundleManager.IsExtensionBundleConfigured())
+            var configured = extensionBundleManager.IsExtensionBundleConfigured();
+            ColoredConsole.WriteLine("extensionBundleManager.IsExtensionBundleConfigured() = " + configured);
```

Comment:
Will users see this message in their terminal? Should this be behind the verbose flag?

Reply:
Sorry this is just for me debugging the issue, not intended to be merged.